### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by ngmachado** [DAM-Protocol/Stroller-Protocol#40]
We are getting to the point that documentation will be needed to Stroller Protocol.

Any opinions on this? Was thinking on using gitbook to create a doc based website to the protocol and "how to".

**Comment by rashtrakoff** [comment]
Agreed. Was thinking about the same. Gitbook will be the best as I am familiar with it and so are most other people.
